src/scripts/install.sh

```diff
@@ -131,20 +98,25 @@ mkdir -p "$rootdir"
 mkdir -p "$batchdir"
 mkdir -p "$mpmdir"
 
-# install mpm
-download "$mpmbaseurl/$mwarch/mpm" "$mpmdir/mpm$binext"
-chmod +x "$mpmdir/mpm$binext"
-
-# install matlab-batch
-download "$batchbaseurl/$mwarch/matlab-batch$binext" "$batchdir/matlab-batch$binext"
-chmod +x "$batchdir/matlab-batch$binext"
-
-# install matlab
-"$mpmdir/mpm$binext" install \
-    --release="$mpmrelease" \
-    --destination="$rootdir" \
-    ${releasestatus} \
-    --products ${PARAM_PRODUCTS} MATLAB
+# Short-circuit if MATLAB & matlab-batch already exist and PARAM_CACHE is true
+if [[ "$PARAM_CACHE" == "1" && -x "$batchdir/matlab-batch$binext" && -x "$rootdir/bin/matlab" ]]; then
```

Comment:
We shouldn't cache `matlab-batch` because it is important that we always make sure we're using the latest version of it. It's a fairly small binary, so just downloading it every time isn't bad.